src/audio/component.c

```diff
@@ -505,3 +505,56 @@ int comp_copy(struct comp_dev *dev)
 
 	return ret;
 }
+
+inline uint32_t get_sample_group_size_in_bytes(const struct ipc4_audio_format fmt)
+{
+	return ((fmt.depth >> 3) * fmt.channels_count);
+}
+
+inline uint32_t get_one_ms_in_bytes(const struct ipc4_audio_format fmt)
```

Comment:
as @lgirdwood suggested - if it's `inline`, you probably meant it to be `static`. I see only 2 functions being added to a header, the rest presumably can be `static`. And I wouldn't use `inline` in .c